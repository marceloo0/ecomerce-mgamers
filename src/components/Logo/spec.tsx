import { screen } from '@testing-library/react'
import { renderWithTheme } from '../../utils/tests/helpers'

import { Logo } from '.'

describe('<Logo />', () => {
  it('should render default', () => {
    const { container } = renderWithTheme(<Logo />)

    expect(screen.getByRole('img', { name: /logo/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
