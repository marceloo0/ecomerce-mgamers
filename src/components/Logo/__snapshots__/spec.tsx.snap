// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render default 1`] = `
.c0 {
  border: none;
  background-color: transparent;
}

.c1 {
  width: 6rem;
}

@media screen and (min-width:768px) and (max-width:1023px) {
  .c1 {
    width: 7rem;
  }
}

@media (min-width:1024px) {
  .c1 {
    width: 9rem;
  }
}

<button
  class="c0"
>
  <img
    alt="logo"
    class="c1"
    role="img"
    src="/img/logo.svg"
  />
</button>
`;
